src/ModelFactory.php

```diff
@@ -54,7 +54,14 @@ public function __construct()
      */
     final public static function findOrCreate(array $attributes): Proxy
     {
-        if ($found = static::repository()->find($attributes)) {
+        // remove attribute prefixes (like "force:" and "optional:")
```

Comment:
I'm thinking this logic should live inside `Zenstruck\Foundry\Instantiator` as this is where the attribute prefix logic lives. Probably a static `normalizeAttributes(array $attributes): array` method. WDYT?